Leanplum-SDK/Classes/LPMessageTemplates.m

```diff
@@ -230,89 +131,10 @@ + (UIViewController *)visibleViewController
-                             [LPActionArg argNamed:LPMT_ARG_ACCEPT_TEXT withString:LPMT_DEFAULT_YES_BUTTON_TEXT],
-                             [LPActionArg argNamed:LPMT_ARG_CANCEL_TEXT withString:LPMT_DEFAULT_NO_BUTTON_TEXT],
-                             [LPActionArg argNamed:LPMT_ARG_ACCEPT_ACTION withAction:nil],
-                             [LPActionArg argNamed:LPMT_ARG_CANCEL_ACTION withAction:nil],
-                             ]
-             withResponder:^BOOL(LPActionContext *context) {
-                 @try {
-                     if (NSClassFromString(@"UIAlertController")) {
-                         UIAlertController *alert = [UIAlertController alertControllerWithTitle:NSLocalizedString([context stringNamed:LPMT_ARG_TITLE], nil) message:NSLocalizedString([context stringNamed:LPMT_ARG_MESSAGE], nil) preferredStyle:UIAlertControllerStyleAlert];
-                         UIAlertAction *cancel = [UIAlertAction actionWithTitle:NSLocalizedString([context stringNamed:LPMT_ARG_CANCEL_TEXT], nil) style:UIAlertActionStyleCancel handler:^(UIAlertAction *action) {
-                             [self alertDismissedWithButtonIndex:0];
-                         }];
-                         [alert addAction:cancel];
-                         UIAlertAction *accept = [UIAlertAction actionWithTitle:NSLocalizedString([context stringNamed:LPMT_ARG_ACCEPT_TEXT], nil) style:UIAlertActionStyleDefault handler:^(UIAlertAction *action) {
-                             [self alertDismissedWithButtonIndex:1];
-                         }];
-                         [alert addAction:accept];
-
-                         [[LPMessageTemplatesClass visibleViewController]
-                          presentViewController:alert animated:YES completion:nil];
-                     } else
-                     {
-                         UIAlertView *alert = [[UIAlertView alloc]
-                                               initWithTitle:NSLocalizedString([context stringNamed:LPMT_ARG_TITLE], nil)
-                                               message:NSLocalizedString([context stringNamed:LPMT_ARG_MESSAGE], nil)
-                                               delegate:self
-                                               cancelButtonTitle:NSLocalizedString([context stringNamed:LPMT_ARG_CANCEL_TEXT], nil)
-                                               otherButtonTitles:NSLocalizedString([context stringNamed:LPMT_ARG_ACCEPT_TEXT], nil),nil];
-                         [alert show];
-                     }
-                     [self->_contexts addObject:context];
-                     return YES;
-                 }
-                 @catch (NSException *exception) {
-                     LOG_LP_MESSAGE_EXCEPTION;
-                     return NO;
-                 }
-             }];
-    
+    [[[LPAlertMessageTemplate alloc] init] defineActionWithContexts:_contexts];
```

Comment:
maybe `[LPAlertMessageTemplate new] defineActionWithContexts:_contexts];` ?

Reply:
`[Class new]` is conventionally old Obj-C so I don't use that